core/platform/memcache/gae_memcache_services.py

```diff
@@ -75,7 +75,7 @@ def delete_multi(keys):
     """Deletes multiple keys in memcache.
 
     Args:
-        keys: dict(str). the keys (strings) to delete.
+        keys: dict. The keys (strings) to delete.
```

Comment:
Are you sure it's a dict? How did you figure this out? What is the format of the dict?